packages/treasury/bundles/install-on-chain.js

```diff
@@ -100,8 +100,8 @@ export async function installOnChain({ agoricNames, board, chainTimerService, na
     [instanceAdmin, treasuryUiDefaults.AMM_NAME, ammInstance],
     [brandAdmin, 'TreasuryGovernance', govBrand],
     [issuerAdmin, 'TreasuryGovernance', govIssuer],
-    [brandAdmin, 'MOE', moeBrand],
-    [issuerAdmin, 'MOE', moeIssuer],
+    [brandAdmin, 'RUN', centralBrand],
```

Comment:
"RUN" should come from config or constant?

Reply:
Done, now a `centralName` property in the function call's options.